### PR TITLE
Upgrade react-native to 0.71.8

### PR DESCRIPTION
**Pull request by yogevbd**

### CODE REVIEW

File: lib/Mock/mocks/NativeCommandsSender.tsx [hunk 1]
```diff
@@ -47,6 +47,7 @@ export class NativeCommandsSender {
 
   pop(commandId: string, componentId: string, _options?: object) {
     return new Promise((resolve) => {
+      if (!LayoutStore.getLayoutById(componentId)) return;
```
Comment (SudoPlz) [inline lib/Mock/mocks/NativeCommandsSender.tsx:50]: curious, what's the use-case here? Why does this return silently? (should we consider throwing an error/warning instead?

**Review by SudoPlz — COMMENTED** [review]
this is shaping up, but we're actually duplicating work here, I also made those changes on another branch. 

That's totally fine though since you picked it up.
Btw, I left 1 comment up there (a question) and reviewed the rest of the PR.
This looks good.

File: lib/Mock/mocks/NativeCommandsSender.tsx (hunk 1 above)
Comment (yogevbd) [reply lib/Mock/mocks/NativeCommandsSender.tsx]: Fixed 👍

File: playground/android/app/build.gradle [hunk 2]
```diff
@@ -1,19 +1,12 @@
...
-        bundleInBeta: true,
-        enableHermes: true,
-        hermesFlagsDebug:['-Xes6-proxy','-output-source-map'],
-        hermesFlagsRelease:['-output-source-map'],
-        hermesCommand: "../../../node_modules/react-native/sdks/hermesc/%OS-BIN%/hermesc",
-]
-
-apply from: "../../../node_modules/react-native/react.gradle"
+react {
+     root = file("../../../")
```
Comment (SudoPlz) [inline playground/android/app/build.gradle:5]: is that all we need?

What about?

```
react = {
    root = file("../../../")
    entryFile = file("index.js")
    cliPath = file("node_modules/react-native/cli.js")
    bundleAssetName = "index.android.bundle"
    bundleInAlpha = true
    bundleInBeta = true
    enableHermes = true
    hermesFlagsDebug = ['-Xes6-proxy','-output-source-map']
    hermesFlagsRelease = ['-output-source-map']
    hermesCommand = file("../../../node_modules/react-native/sdks/hermesc/%OS-BIN%/hermesc")
}
```
Do we not care about the other values that used to be there?

File: playground/android/app/build.gradle [hunk 3]
```diff
@@ -1,19 +1,12 @@
...
+}
+
+
+def jscFlavor = 'org.webkit:android-jsc:+'
 
 android {
     compileSdkVersion rootProject.ext.get("compileSdkVersion")
```
Comment (SudoPlz) [inline playground/android/app/build.gradle:12]: consider adding a namespace here, like so:

`namespace "com.reactnativenavigation.playgroun` and remove `package="com.reactnativenavigation.playground"` from `AndroidManifest.xml` for the package name.

File: playground/android/app/build.gradle [hunk 4]
```diff
@@ -57,20 +50,20 @@ dependencies {
     implementation 'com.google.android.material:material:1.4.0'
```
Comment (SudoPlz) [inline playground/android/app/build.gradle:50]: I think we can safely delete the line above the one I highlighted:

`implementation fileTree(dir: 'libs', include: ['*.jar'])`

File: playground/android/build.gradle [hunk 5]
```diff
@@ -6,9 +6,9 @@ buildscript {
         RNNKotlinVersion = kotlinVersion
         detoxKotlinVersion = kotlinVersion
         compileSdkVersion = 33
-        buildToolsVersion = "31.0.0"
+        buildToolsVersion = "33.0.0"
         minSdkVersion = 21
-        targetSdkVersion = 31
+        targetSdkVersion = 33
     }
```
Comment (SudoPlz) [inline playground/android/build.gradle:12]: We may want to specify an ndk version here:

```suggestion
        // We use NDK 23 which has both M1 support and is the side-by-side NDK version from AGP.
        ndkVersion = "23.1.7779620"
    }
```

File: playground/android/build.gradle [hunk 6]
```diff
@@ -18,26 +18,18 @@ buildscript {
 
     dependencies {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
-        classpath 'com.android.tools.build:gradle:7.1.1'
+        classpath("com.facebook.react:react-native-gradle-plugin")
+        classpath 'com.android.tools.build:gradle:7.3.1'
         classpath("de.undercouch:gradle-download-task:5.0.1")
```
Comment (SudoPlz) [inline playground/android/build.gradle:23]: we probably no longer need this `classpath("de.undercouch:gradle-download-task:5.0.1")`

File: playground/android/app/build.gradle (hunk 2 above)
Comment (yogevbd) [reply playground/android/app/build.gradle:5]: I think those are RN's defaults

**Review by SudoPlz — APPROVED** [review]
LGTM

**Pull request merged**